tests/inference_on_device_test.py

```diff
@@ -430,11 +430,13 @@ def test_embedding_nets_integration_training_device(
 
         posterior = inference.build_posterior(
             density_estimator_train,
-            **{}
-            if inference_method == SNPE_A
-            else dict(
-                mcmc_method="slice_np_vectorized",
-                mcmc_parameters=dict(thin=10, num_chains=20, warmup_steps=10),
+            **(
```

Comment:
this looks like a formatting change we did not have before. Which ruff version are you using?

Reply:
I am using v0.3.3 (latest). There is also a related PR #1010 which I believe you merged already, so we should be fine.

Reply:
I just re-ran the check and the only errors I get are from the tutorials

```
❯ ruff check .
tutorials/02_flexible_interface.ipynb:cell 8:1:1: I001 [*] Import block is un-sorted or un-formatted
tutorials/03_multiround_inference.ipynb:cell 4:1:1: I001 [*] Import block is un-sorted or un-formatted
tutorials/05_embedding_net.ipynb:cell 4:1:1: I001 [*] Import block is un-sorted or un-formatted
tutorials/07_conditional_distributions.ipynb:cell 28:1:1: I001 [*] Import block is un-sorted or un-formatted
tutorials/14_iid_data_and_permutation_invariant_embeddings.ipynb:cell 3:1:1: I001 [*] Import block is un-sorted or un-formatted
Found 5 errors.
```